Jenkinsfile

```diff
@@ -22,15 +24,33 @@ pipeline {
         sh 'docker pull govukpay/postgres:9.4.4'
         sh 'mvn clean package'
       }
+      post {
+        failure {
+          metricsUtils("adminusers.maven-build.failure", 1, "new")
+        }
+        success {
+          script {
+            def long mavenBuildDuration = (System.currentTimeMillis() - currentBuild.startTimeInMillis) / 1000
```

Comment:
Is it worth abstracting this to a function. Like 
`metricsForSuccessfulBuild microservice build-type` ?

Reply:
Yes it would. I think I did the metrics implementation 20 different ways with functions, without, with classes etc and spent a lot of time trying to wrestle with groovy and jenkins that I missed this when I finally got it working.